src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((result, style) => {
```

Comment:
The use of 'reduce' here is appropriate as it transforms an array into a single output value (in this case, an object). However, ensure that the code handles cases where there might be empty or malformed style declarations that could result in an undefined property or value.